research/fedbn/jobs/client_api/app/config/config_fed_client.conf

```diff
@@ -0,0 +1,77 @@
+{
```

Comment:
since you already provide the script to create job, we don't need to check-in this created job.